themes/bootstrap3/js/facets.js

```diff
@@ -297,9 +298,21 @@ VuFind.register('multiFacetsSelection', function multiFacetsSelection() {
     toggleSelectedFacetStyle(elem);
   }
 
+  function saveUserSelectionLastState(state) {
+    localStorage.setItem(local_storage_variable_name, state);
```

Comment:
Noticed one more thing: While this may work, I think it would be more appropriate to set the value explicitly to `state ? 'true' : 'false'`) since `setItem` takes a string but `state` is a boolean.